docs/instructor-training-workflow.txt

```diff
@@ -0,0 +1,211 @@
+
+## Setting up training course.
+
+* [admin creates training event]: and assigns people to it.
+  - Admin creates new <Event>.
+  - Admin goes to {training queue display}.
+  - Admin selects <Event>.
+  - Admin selects some <Training Application>s.
+  - AMY creates <Task>s to add all the selected <Person>s to the <Event>.
+  - AMY connects the <Training Application>s to the <Event>.
+  - AMY moves <Training Application>s to <Assigned Application List>.
+  - AMY provides single-click "send to all" email link to admin to inform trainees.
+  - AMY creates <Training Progress> record for each <Person>.
+
+## Administering training course applications.
+
+* [admin removes people from training event][a]: and discards applications
+  - Admin goes to {assigned training application display}.
+  - Admin selects <Training Application>s.
+  - Admin removes them from training event.
+  - AMY deletes associated <Task>s.
+  - AMY deletes <Training Application>s.
+
+* [admin removes people from training event][b]: but keeps applications
+  - Admin goes to {assigned training application display}.
+  - Admin selects <Training Application>s.
+  - Admin removes them from training event.
+  - AMY deletes associated <Task>s.
+  - AMY moves <Training Application>s back to <Training Application List>.
+
+## Reviewing training progress.
+
+* [admin views training progress]: to see who's done what.
+  - Admin views {training progress page}.
+  - Admin selects filtering and sorting criteria (if any).
+
+## Tracking submission of written work.
+
+* [trainee reports submission of written work]: such as pull request or comment on lesson.
+  - Trainee views their own {training progress}.
+  - Trainee enters URL of written work.
+  - AMY updates <Training Task> associated with <Person> and <Event>.
+  - AMY adds <Training Progress> to <Training Progress Review List>.
+
+* [admin reports submission of written work]: on behalf of trainee.
+  - Admin looks up <Training Progress> associated with trainee.
+  - Admin enters URL of written work in <Training Task>.
+
+* [admin reports submission of written work]: on behalf of trainee.
+  - Admin looks up <Training Progress> associated with trainee.
+  - Admin enters URL of written work in <Training Task>.
+  - AMY adds <Training Progress> to <Training Progress Review List>.
+
+* [admin reports completion of written work]: on behalf of trainee.
+  - Admin looks up <Training Progress> associated with trainee.
+  - Admin enters URL of written work in <Training Task>.
+  - Admin marks <Training Task> as complete.
+
+## Tracking participation in interactive work.
+
+* [admin reports successful completion of interactive work]: such as discussion session or lesson demo.
+  - Admin views {training progress} for trainee.
+  - Admin selects <Training Task>.
+  - Admin:
+    - Enters date of interactive event.
+    - Adds comments (if any).
+  - Admin indicates whether <Training Task> is now complete.
```

Comment:
This requires Mentors and Examiners to go to the {training progress} view *for each trainee*. If I leaded a session, I'd like to have a view of all trainees, select trainees that attended my session, and then click "bulk pass".